Solutions/Network Session Solution for Microsoft Sentinel/Hunting Queries/MismatchBetweenDestinationAppNameAndDestinationPort.yaml

```diff
@@ -0,0 +1,84 @@
+id: a61e9fc1-dd9e-4588-a497-2a34a1d811bf
+name: Mismatch between Destination App name and Destination Port (ASIM Network Session schema)
+description: |
+  'Every standard app has a port associated with it. This query will identify if destination port associated with destination app is not standard which can be a case of network spoofing attack.'
+
+tags:
+  - Schema: ASimNetworkSessions
+    SchemaVersion: 0.2.4
+requiredDataConnectors:
+  - connectorId: AWSS3
+    dataTypes:
+      - AWSVPCFlow
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceNetworkEvents
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+  - connectorId: WindowsForwardedEvents
+    dataTypes:
+      - WindowsEvent
+  - connectorId: Zscaler
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: MicrosoftSysmonForLinux
+    dataTypes:
+      - Syslog
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: AzureMonitor(VMInsights)
+    dataTypes:
+      - VMConnection
+  - connectorId: AzureFirewall
+    dataTypes:
+      - AzureDiagnostics
+  - connectorId: AzureNSG
+    dataTypes:
+      - AzureDiagnostics
+  - connectorId: CiscoASA
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL
+  - connectorId: AIVectraStream
+    dataTypes:
+      - VectraStream
+  - connectorId: CheckPoint
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: Fortinet
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: CiscoMeraki
+    dataTypes:
+      - Syslog
+      - CiscoMerakiNativePoller
+
+tactics:
+  - Discovery
+
+query: |
+  _Im_NetworkSession(starttime=ago(1d),endtime=now())
```

Comment:
Same issue on this one:
![image](https://user-images.githubusercontent.com/45466083/206034295-d26abd44-1dca-47e8-ae07-be4f6dd2d8c7.png)

Reply:
Resolving per comment - https://github.com/Azure/Azure-Sentinel/pull/6607#issuecomment-1340405681